crates/swift-integration-tests/src/tuple.rs

```diff
@@ -0,0 +1,34 @@
+#[swift_bridge::bridge]
+mod ffi {
+    extern "Rust" {
+        type TupleTestOpaqueRustType;
+        #[swift_bridge(init)]
+        fn new(val: i32) -> TupleTestOpaqueRustType;
+        fn val(&self) -> i32;
+        fn reflect_tuple_primitives(tuple: (i16, u32)) -> (i16, u32);
```

Comment:
Minor note:

I like to prefix freestanding functions with `rust_`, such as `rust_reflect_tuple_primitives`.

This makes it easier to quickly recognize which functions are hosted on the Rust side vs. the Swift side.

Not required, just sharing an idea.